selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/DashboardWorkspace.java

```diff
@@ -917,6 +917,7 @@ public void checkAgentExists(String agentName) {
   }
 
   public void switchAgentState(String agentName) {
+    loader.waitOnClosed();
```

Comment:
IMHO it's better to don't link method itself with with wait on clean up after the previous operation, that is wait until loader from previous operation closes.
To make **switchAgentState()** more universal, it's better to move `loader.waitOnClosed()` to the caller method.